server/apps/client.go

```diff
@@ -41,7 +42,16 @@ func (s *Service) PostWish(call Call) (*CallResponse, error) {
 	if err != nil {
 		return nil, err
 	}
-	resp, err := s.post(app, call.Data.Context.ActingUserID, call.Wish.URL, call.Data)
+
+	url := call.Wish.URL
+	if !strings.HasPrefix(url, "https://") && !strings.HasPrefix(url, "http://") {
+		if !strings.HasPrefix(url, "/") {
+			url += "/"
+		}
+		url = app.Manifest.RootURL + url
```

Comment:
If the url passed in is a relative URL, we must mount it onto the app's root URL

Reply:
@levb I see this fix didn't make it into master. I also realized that I have a typo here, as it should have been `url = "/" + url`

How do we handle the case when a relative URL is used in the `call.FormURL`?